utils/data-file.c

```diff
@@ -357,51 +357,44 @@ static void check_data_order(struct ftrace_file_handle *handle)
 		pr_dbg("bitfield order is different!\n");
 }
 
-#define RECORD_MSG  "Was '%s' compiled with -pg or\n"		\
-"\t-finstrument-functions flag and ran with uftrace record?\n"
-
```

Comment:
Why did you remove this message?

Reply:
The previous usage of RECORD_MSG was as follows:
```
$ uftrace info
WARN: cannot open data: uftrace.data: No such file or directory

$ uftrace info a.out 
WARN: Was 'a.out' compiled with -pg or
        -finstrument-functions flag and ran with uftrace record?
WARN: cannot open data: uftrace.data: No such file or directory

$ uftrace a.out 
Hello World!
# DURATION    TID     FUNCTION
   1.053 us [118894] | __monstartup();
   0.853 us [118894] | __cxa_atexit();
            [118894] | main() {
   6.146 us [118894] |   puts();
   3.757 us [118894] |   fflush();
  23.340 us [118894] |   malloc();
   0.826 us [118894] |   free();
  35.827 us [118894] | } /* main */
```
In this case, I thought the message is not informative or correct so removed it.

Reply:
Fair enough. :)